praatio/audio.py

```diff
@@ -30,73 +31,138 @@ class AudioDeletion:
 _DELETE: Final = "delete"
 
 ZERO_CROSSING_TIMESTEP: Final = 0.002
+DEFAULT_SINE_FREQUENCY = 200
 
 
-def samplesAsNums(waveData, sampleWidth: int) -> Tuple[int, ...]:
-    """Convert samples of a python wave object from bytes to numbers"""
-    if len(waveData) == 0:
+def calculateMaxAmplitude(sampleWidth: int) -> int:
+    """Gets the maximum possible amplitude for a given sample width"""
+    return 2 ** (sampleWidth * 8 - 1) - 1
```

Comment:
Perhaps this could use some comments, because there are magic numbers that make it harder to understand.

My understanding is that the maximum amplitude is `2^(n - 1)`, where `n` is the number of bits. So I'm guessing this means `sampleWidth` is in bytes, and that's why you multiply by `8`?

The only part I don't understand is the final `- 1`, but that could just be my own ignorance.

Reply:
I added some documentation which I hope will help! [09430c2](https://github.com/timmahrt/praatIO/pull/41/commits/09430c27df16ee980dd2a784eea6358102dbe7aa)